uk.co.compucorp.civicrm.hrleaveandabsences/js/angular/src/leave-absences/shared/controllers/request-ctrl.js

```diff
@@ -409,10 +420,14 @@ define([
           //get a clone so self it is not the same reference as passed from callee
```

Comment:
`//get a clone so self it is not the same reference as passed from callee`
This sentence does not sound correct to me, could you fix it?